### PR TITLE
The file downloaded is regular zip format, not gzip

### PR DESCRIPTION
**Pull request by dav**
so the `gzip -d` command was barfing.

This switches to using `unzip` instead.

I'm not sure if previously the file was actually a `.gz` misnamed as `.zip` and now it is actually a `.zip` or what.

@arielf making this a PR because wanted to see if you saw a similar issue on linux?

### CODE REVIEW

**Comment by arielf** [comment]
This is a good catch.
It works for me without the change, but this change can't hurt. Thank you.

The file that's downloaded is a zip file (not a GNU gzip file).

The `-O ....gz` is there to make `gzip` not complain about it. Without it, `gzip` complains, e.g:

   `gzip: z: unknown suffix -- ignored`

I didn't hit it because (recent enough versions of?) `gzip` know about `zip` format (by looking at the content rather than the extension), and do the right thing.

```
7:39 xxxx 452> wget http://mattmahoney.net/dc/text8.zip -O z.gz
--2016-07-01 17:39:16--  http://mattmahoney.net/dc/text8.zip
Resolving mattmahoney.net (mattmahoney.net)... 98.139.135.129
Connecting to mattmahoney.net (mattmahoney.net)|98.139.135.129|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 31344016 (30M) [application/zip]
Saving to: 'z.gz'

z.gz                   100%[===================>]  29.89M  1.86MB/s    in 16s

2016-07-01 17:39:32 (1.91 MB/s) - 'z.gz' saved [31344016/31344016]

17:39 xxxx 453> file z.gz
z.gz: Zip archive data, at least v2.0 to extract

17:39 xxxx 454> cp  z.gz  z.save.gz

17:40 xxxx 457> gzip -d z.gz

# all is good here.  text8 is extracted fine

17:40 go 458> unzip z.save.gz
Archive:  z.save.gz
  inflating: text8

# Also seems to work fine
```

Bottom line: switching to `unzip` can't hurt (on Ubuntu).
